pkg/defaultmonitortests/types.go

```diff
@@ -122,6 +123,7 @@ func newDefaultMonitorTests(info monitortestframework.MonitorTestInitializationI
 
 	monitorTestRegistry.AddMonitorTestOrDie("monitoring-statefulsets-recreation", "Monitoring", statefulsetsrecreation.NewStatefulsetsChecker())
 	monitorTestRegistry.AddMonitorTestOrDie("metrics-api-availability", "Monitoring", disruptionmetricsapi.NewAvailabilityInvariant())
+	monitorTestRegistry.AddMonitorTestOrDie(clienttest.MonitorName, "API Unreachable", clienttest.NewAPIUnReachableFromClientMonitorTest())
```

Comment:
Second string param should be a component name. kube-apiserver looks to be used above so this might be best here.